src/app/(main)/interior-decorating-vs-interior-design/page.tsx

```diff
@@ -0,0 +1,158 @@
+import type { Metadata } from 'next';
+import Image from 'next/image';
+
+import BarChartImage from './bar-chart.png';
+import HeroImage from './hero.jpg';
+import LineGraphImage from './line-graph.png';
+import SalaryCompareImage from './salary-compare-bg.jpg';
+import TrainingRequirementMobileImage from './training-requirement-bg-mobile.jpg';
+import TrainingRequirementImage from './training-requirement-bg.jpg';
+import type { PageComponent } from '@/app/serverComponent';
+import { BackgroundImage } from '@/components/backgroundImage';
+import { GetStartedSection } from '@/components/getStartedSection';
+import PaletteIcon from '@/components/icons/palette.svg';
+import ScreenshotIcon from '@/components/icons/screenshot.svg';
+
+export const metadata: Metadata = {
+  title: 'How It Works',
```

Comment:
The tile should be "Interior Decorating vs. Interior Design", I think.

And we should add a meta description. Can you ask Kayla for one?